### PR TITLE
Update gradle.yml

### PR DESCRIPTION
**Pull request by DijonnaWatson**
updating the gradle.yml to generate the .proto autogenerated gRpc and java. We all need this for our branches.

### CODE REVIEW

**Comment by DijonnaWatson** [comment]
So far, I think I'm done with step 2 on the assignment

File: src/clientandserver/FactorServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      }
+
+      // Translate response back into external type
+      coordinatorResponse externalResponse =
+          coordinatorResponse.newBuilder().setStatus(grpcStatus).build();
+
+      responseObserver.onNext(externalResponse); // sending the byte back to the
+                                                 // Client on the open socket
+      responseObserver.onCompleted();
+    }
```
Comment (jeremy23newman) [inline src/clientandserver/FactorServiceImpl.java:93]: good job with the service implementation

File: protobuf/dataStore.proto [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package apiProto;
+
+//coordinator to dataStore
```
Comment (jeremy23newman) [inline protobuf/dataStore.proto:3]: good job generating the proto files looks good to me

**Review by jeremy23newman — APPROVED** [review]

File: protobuf/coordinatorEngine.proto [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	enum FactorComputeResult {
+		success = 1;
+		failure = 2;
+	}
+	optional FactorComputeResult status = 3;
+}
+
+service FactorService{
+	rpc compute (coordinatorRequest) returns (coordinatorResponse);
+}
```
Comment (ViceRaven) [inline protobuf/coordinatorEngine.proto:20]: I see the approach you took for the coordinator engine. It makes sense using an enum. Nicely done.

**Pull request merged**